iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifierTest.java

```diff
@@ -49,7 +49,7 @@ class KubernetesVerifierTest {
 
   @Test
   void verifierShouldSucceedOnHelmProjectWithSecondaryLocationsRaised() {
-    TextRange expectedShiftedTextRangePrimaryIssue = range(7, 0, 7, 40);
+    TextRange expectedShiftedTextRangePrimaryIssue = range(7, 22, 8, 0);
```

Comment:
How reasonable is that the range is shifted onto the next line? Or is it something to be fixed by Go AST adjustments?